src/fsharp/fsi/fsi.fs

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-module Microsoft.FSharp.Compiler.Interactive.Shell
```

Comment:
In general you should undo all the changes in this file. Some additional changes _may_ be needed to get it to compile on .NET Core. However this file has largely diverged from Microsoft\visualfsharp (or at least the changes here have not yet been folded back into that repo).  So we effectively have to work out how to compile this on .NET Core.  Alternatively remove it from the .netcore version of FSharp.Compiler.Service to start, depending on whether you need dynamic compilation or not.